webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyStatePattern.js

```diff
@@ -1,29 +1,65 @@
 import React from 'react';
-import { EmptyState as PfEmptyState } from 'patternfly-react';
+import { Icon } from 'patternfly-react';
+import {
+  Title,
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateBody,
+  EmptyStateSecondaryActions,
+} from '@patternfly/react-core';
 import { emptyStatePatternPropTypes } from './EmptyStatePropTypes';
+import { translate as __ } from '../../../common/I18n';
+import './EmptyState.scss';
 
 const EmptyStatePattern = props => {
   const {
     documentation,
     action,
     secondaryActions,
-    icon,
     iconType,
+    icon,
     header,
     description,
   } = props;
 
+  const DocumentationBlock = () => {
+    if (!documentation) {
+      return null;
+    }
+    if (React.isValidElement(documentation)) {
```

Comment:
Can you add a small comment above, e.g:
"The documentation prop can be also a customized node"